tests/test_models.py

```diff
@@ -82,7 +94,7 @@ def test_serialize_a_shopcart_datestring(self):
         """ Test serialization of a Shopcart with date string """
         date_time = datetime.now()
         date_time_str = datetime.isoformat(date_time)
-        shopcart = Shopcart(user_id=101, create_time=date_time_str, update_time=date_time_str)
+        shopcart = Shopcart(user_id=101, create_time=date_time, update_time=date_time)
```

Comment:
Just wondering why you changed this here? This is a test to make sure the serialize will work with string input for create_time and update_time. Otherwise this function is exactly the same as test_serialize_a_shopcart

Reply:
as in the definition of shopcart, create_time and update_time should be type of datetime not str?

Reply:
> Just wondering why you changed this here? This is a test to make sure the serialize will work with string input for create_time and update_time. Otherwise this function is exactly the same as test_serialize_a_shopcart

sorry for the undiscussed change, as my IDE reports this as errors. if you need this test this case, i will change it back

Reply:
hey, sorry can you change it so the string is used in def test_serialize_a_shopcart_datestring